superset/charts/schemas.py

```diff
@@ -162,6 +162,7 @@ class ChartEntityResponseSchema(Schema):
     certification_details = fields.String(
         metadata={"description": certification_details_description}
     )
+    thumbnail_url = fields.String(allow_none=True)
```

Comment:
I noticed that `thumbnail_url` was missing on the Marshmallow schema despite being returned by the API